src/MSAL.PCL/ClientApplicationBase.cs

```diff
@@ -134,7 +135,7 @@ public IEnumerable<User> Users
         /// <returns></returns>
         public async Task<AuthenticationResult> AcquireTokenSilentAsync(string[] scope)
         {
-            Authenticator authenticator = new Authenticator(this.Authority, this.ValidateAuthority, this.CorrelationId);
+            Authority authenticator = Internal.Instance.Authority.CreateAuthority(this.Authority, this.ValidateAuthority);
```

Comment:
update it across files :)

---
In reply to: [77229056](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/116#discussion_r77229056) [](ancestors = 77229056)